pages/linux/f5fpc.md

```diff
@@ -0,0 +1,23 @@
+# f5fpc
+
+> A proprietry commercial SSL VPN client by BIG-IP Edge.
+
+- Open a new VPN connection:
+
+`f5fpc --start`
+
+- Open a new VPN connection to a specific host:
+
+`f5fpc --start --host {{host.example.com}}` 
+
+- Specify a username:
```

Comment:
I guess it prompts you for the password ? Should we mention it here ?

Reply:
Ah, good idea.